<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mean_curvature_flow_skeleton_plugin</class>
 <widget class="QDockWidget" name="Mean_curvature_flow_skeleton_plugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="is_medially_centered">
      <property name="text">
       <string>Is medially centered</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_omega_H">
            <property name="text">
             <string>quality_speed_tradeoff</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="omega_H"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_omega_P">
            <property name="text">
             <string>medially_centered_speed_tradeoff</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="omega_P"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_min_edge_length">
            <property name="text">
             <string>min_edge_length</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="min_edge_length"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_delta_area">
            <property name="text">
             <string>area_variation_factor </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="delta_area"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_run">
          <property name="text">
           <string>Run one iteration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_converge">
          <property name="text">
           <string>Run to convergence</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_skeletonize">
          <property name="text">
           <string>Skeletonize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_segment">
          <property name="text">
           <string>Segment</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Steps of an iteration:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_contract">
        <property name="text">
         <string> 1 - Contract</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_collapse">
        <property name="text">
         <string>2 - Collapse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_split">
        <property name="text">
         <string>3 - Split</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_degeneracy">
        <property name="text">
         <string>4 - Degeneracy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>contract()</slot>
 </slots>
</ui>
